rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/OpenehrVersionedEhrStatusController.java

```diff
@@ -106,6 +107,9 @@ public OpenehrVersionedEhrStatusController(EhrService ehrService, ContributionSe
     }
 
     @GetMapping(path = "/version")
+    // checkAbacPre /-Post attributes (type, auth object, subject, payload, content type)
+    @PreAuthorize("checkAbacPre(@openehrVersionedEhrStatusController.EHR_STATUS, authentication, "
```

Comment:
This could be simplified based on the comment in `CustomMethodSecurityExpressionRoot`